lib/cose/keys.ex

```diff
@@ -36,3 +60,38 @@ defmodule COSE.Keys.OKP do
     :crypto.verify(:eddsa, :sha256, to_be_verified, signature, [ver_key.x, :ed25519])
   end
 end
+
+defmodule COSE.Keys.ECDSA do
```

Comment:
Any reason for not following defining the key struct here, as in the rest of the library?